ValheimPlus/GameClasses/ZNet.cs

```diff
@@ -66,6 +50,28 @@ private static void Postfix(ref ZNet __instance)
                 ZRoutedRpc.instance.InvokeRoutedRPC(ZRoutedRpc.instance.GetServerPeerID(), "VPlusConfigSync", new object[] { new ZPackage() });
             }
         }
+
+        /// <summary>
+        /// Alter server player limit
+        /// </summary>
+        [HarmonyTranspiler]
+        public static IEnumerable<CodeInstruction> Transpiler(IEnumerable<CodeInstruction> instructions)
+        {
+            List<CodeInstruction> il = instructions.ToList();
+
+            for (int i = 0; i < il.Count; i++)
+            {
+                if (il[i].Calls(method_ZNet_GetNrOfPlayers))
+                {
+                    il[i + 1].operand = Configuration.Current.Server.maxPlayers;
+                    return il.AsEnumerable();
+                }
+            }
+
+            ZLog.LogError("Failed to apply ZNet.RPC_PeerInfo.Transpiler");
```

Comment:
this is true, but it's very technical. 
```suggestion
            ZLog.LogError("Failed to alter server player limit (ZNet.RPC_PeerInfo.Transpiler)");
```